FaultManagerApi.yaml

```diff
@@ -4,12 +4,12 @@ info:
   description: 'API to retrieve the latest processed fault data from FRAIHMWORK'
   contact:
     email: g.dorchies@resilienx.com
-  version: 0.0.1
+  version: 0.0.2
 externalDocs:
   description: Find out more about Swagger
   url: http://swagger.io  
 servers:
-- url: https://<server>.fraihmwork.resilienx.com/api/faults/v0/
+- url: https://<server>.fraihmwork.resilienx.com/api/active/faults/v0/
 tags:
 - name: Fault Data Retrieval
```

Comment:
If you make the tag, you should probably use it below (all the paths below use "Fault Entity")

Reply:
Will rename to ActiveFaultApi.yaml. Updated to consolidate use of "Fault Data Retrieval" tag.